analyzers/src/ITs.JsonParser/Json/SarifIssues.cs

```diff
@@ -0,0 +1,54 @@
+﻿namespace ITs.JsonParser.Json;
+
+// Format of JSONs in "output/"
+internal class SarifIssues
```

Comment:
Do you mean it should be public?
I think it makes more sense to be internal, especially for the UTs.
The UTs right now do not use this, the only thing used is the `Execute` entrypoint, which is public.

And IMO that's the only thing that should be used, since I don't think making these by hand on the UTs is a good thing, I prefer making raw JSON files and running the entire read-map-write "pipeline".

Reply:
Yes, everything should be `public`. There's no practical reason to be `internal` in this case. Having `internal` on half of the files will lead to inconsistency